### ISSUE TITLE
Use enable-stack-protector

### ISSUE THREAD

**Issue by asieira** [sgerrand/docker-glibc-builder#17]
The following is available since glibc 2.25:
```
* Most of glibc can now be built with the stack smashing protector enabled.
  It is recommended to build glibc with --enable-stack-protector=strong.
  Implemented by Nick Alcock (Oracle).
```

I would like to suggest that https://github.com/sgerrand/docker-glibc-builder/blob/master/builder is updated to include this parameter when running `configure`.

**Comment by frezbo** [comment]
@asieira thanks for the information, I will update all my local builds.

**Comment by sgerrand** [comment]
Thanks for suggesting this change. I've implemented it in ec40a5748b1665aff7d72d43454c8177eed505d5.
